src/Form/Input.js

```diff
@@ -28,29 +29,69 @@ const StyledInput = createComponent({
     transition: 250ms all;
     -webkit-appearance: none;
     font-family: inherit;
-    font-size: ${theme.fontSizes[size]}px;
+    font-size: ${themeGet('typography.fontSize')}px;
+    color: ${theme.colors.greyDarkest};
 
     &:hover,
-    &:focus,
     &:active {
       border-color: ${theme.colors.greyDark};
     }
 
+    &:focus {
+      border-color: ${theme.colors.primary};
+    }
+
     ::placeholder {
-      color: ${theme.colors.greyDark};
+      color: ${theme.colors.grey};
     }
 
     &[disabled] {
-      opacity: 0.65;
+      background-color: ${theme.colors.white};
+      border-color: ${theme.colors.greyLight};
+      color: ${theme.colors.grey};
     }
 
     ${isFloating &&
       css`
-        padding-bottom: 0;
+        line-height: 14px;
+        padding-top: 14px;
+        padding-bottom: 0px;
+      `};
+
+    ${leftIcon &&
+      css`
+        padding-left: ${(leftIconProps && leftIconProps.size) || 16 + 12}px;
```

Comment:
Rather than hard coding this padding value, how about using the icon's margin style prop. That way different layouts can be supported without requiring overwriting the input styles.
![Screen Shot 2019-08-21 at 2 11 04 PM](https://user-images.githubusercontent.com/16051172/63468815-9b4b2680-c41d-11e9-879f-54211e372b2d.png)

Reply:
Using the icon's margin wouldn't affect the input though, since it is absolutely positioned. I believe that line isn't working as expected.

Reply:
I mean, using the icon's margin to determine the input's padding, sorry for being unclear.

Reply:
The user should be able to determine that gap in a few ways, here's one within the input itself:
![image](https://user-images.githubusercontent.com/10860014/63473871-aeb9ba00-c43c-11e9-9cc6-2fc7a9c3d721.png)

They should also be able to change that in the `theme.js` file as well since we also have `InputLeftIcon` & `InputRightIcon`.